packages/backend/src/index.ts

```diff
@@ -70,7 +70,7 @@ global.logger = winston.createLogger({
     app.use('/storage', Storage());
     app.use('/auth', Auth());
     app.use('/delivery', Delivery());
-    app.use('/notifications', Notifications());
+    app.use('/notifications', Notifications(getDeliveryServiceProperties));
```

Comment:
DeliveryServiceproperties are already initialized in Line 63 use the value of that var to pass it through the function instead of providing a function